rakefile.netstd.rb

```diff
@@ -0,0 +1,405 @@
+require 'albacore'
+
+msbuild_command = "C:/Program Files (x86)/MSBuild/14.0/Bin/MSBuild.exe"
+if !File.file?(msbuild_command)
+  raise "MSBuild not found"
+end
+
+nuget_command   = ".nuget/nuget.exe"
+gitlink_command = "packages/gitlink.2.3.0/lib/net45/GitLink.exe"
+xunit_command   = "packages/xunit.runner.console.2.0.0/tools/xunit.console.exe"
+
+solution        = "FakeItEasy.netstd.sln"
+assembly_info   = "src/CommonAssemblyInfo.cs"
+version         = IO.read(assembly_info)[/AssemblyInformationalVersion\("([^"]+)"\)/, 1]
+version_suffix  = ENV["VERSION_SUFFIX"]
+repo_url        = "https://github.com/FakeItEasy/FakeItEasy"
+nuspec          = "src/FakeItEasy.netstd/FakeItEasy.nuspec"
+analyzer_nuspec = "src/FakeItEasy.Analyzer/FakeItEasy.Analyzer.nuspec"
+logs            = "artifacts/logs"
+output          = File.absolute_path("artifacts/output")
+tests           = "artifacts/tests"
+packages        = File.absolute_path("packages")
+
+gitlinks        = ["FakeItEasy", "FakeItEasy.Analyzer", "FakeItEasy.netstd"]
+
+unit_tests = [
+  "tests/FakeItEasy.Tests/bin/Release/FakeItEasy.Tests.dll",
+  "tests/FakeItEasy.Analyzer.Tests/bin/Release/FakeItEasy.Analyzer.Tests.dll"
+]
+
+netstd_unit_test_directories = [

```

Comment:
Similar to my other comment, these aren't really netstd tests. They are tests that target any platform which requires them to be run using the `dotnet` CLI. (In our case, the test project is `netcoreapp`.). Perhaps this would be better named `dotnet_unit_test_directories`.